playwright-e2e/pages/dsm/kitUpload-page.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+//Inputting the various kit types that can be uploaded in DSM
+//todo Update as needed as more kit upload tests are created
+enum KitType {
+    BLOOD = 'BLOOD',
```

Comment:
all kits (for rgp) are planned to be blood & rna instead of blood (if they aren't already - will check with Jeff)